html/captive-portal/templates/content-with-choice.html

```diff
@@ -1,9 +1,28 @@
-<h4>Select an option : </h4>
+[% IF content %]
```

Comment:
Would it be possible to have the menu always displayed but when there is a selected option it display smaller at the top.

Otherwise the only way to change option is to click the X but that makes you restart at the beginning so if you're at the Xth step of the portal you have to redo everything if you selected the wrong option